### DISCUSSION TITLE
Add latest Libbi features - Export now, Charging from Grid

### DISCUSSION THREAD

**Discussion by whittyone** [CJNE/ha-myenergi#456] [Ideas]
This integration is awesome; its beyond my capability but It'd be really great if could be extended to allow you to:
- set Libbi to Export Now (And to turn off Export now)
- Enable/Disable Charging from Grid, and set the target charge level.

It'd also be nice to have if you could toggle whether the libbi is allowed to power Eddi and Zappi.

**Comment by G6EJD** [comment]
It would be possible if someone had reverse engineered the Libbi API, some of the features have been established already. MyEnergi doesn’t release API details, so it could be some time.

**Comment by plord12** [comment]
See https://github.com/CJNE/ha-myenergi/issues/442 and https://github.com/CJNE/pymyenergi/issues/18

The "pre-release" from https://github.com/CJNE/ha-myenergi/issues/442#issuecomment-1784222844 has been working for me.

**Reply by whittyone** [reply]
Cool thanks - I don't know how I missed that to be honest; I'll check it out.

**Comment by MalcolmSpencer** [comment]
I see there has been some great work on pymyenergi and this now support various functions on the Libbi. It says that 'This Home Assistant add-on talks to the myenergi API using the [pymyenergi python library]' Therefore is Libbi now supported?

**Comment by whittyone** [comment]
These capabilities are now available in the latest release. V0.0.28
